docs/Usage_example_FDP.ipynb

```diff
@@ -474,14 +342,25 @@
     "from sempyro.dcat import DCATDataset\n",
     "\n",
     "class FDPDataset(DCATDataset):\n",
```

Comment:
HRIDataset zou afdoende moeten zijn

Reply:
HRIDataset is anders dan default shapes op v1.16 FDP